src/main/java/io/cryostat/ConfigProperties.java

```diff
@@ -58,4 +58,6 @@ public class ConfigProperties {
     public static final String SSL_TRUSTSTORE_DIR = "ssl.truststore.dir";
 
     public static final String URI_RANGE = "cryostat.target.uri-range";
+
+    public static final String AGENT_TLS_ENABLED = "cryostat.agent.tls.enabled";
```

Comment:
I think renaming this from `enabled` to `required` would also better indicate what the control is for.